ui/src/js/viewModels/user.js

```diff
@@ -202,8 +204,15 @@ define(['appController','ojs/ojrouter','utils','ojs/ojcore', 'knockout', 'jquery
                                             console.log("User organizations not loaded");
                                             self.userOrgValid(false);
 
+                                        }).then(function () {
+                                            var dbsCheckStates = [
+                                                { value: 'Y', label: 'Yes' },
+                                                { value: 'N', label: 'No' },
+                                                { value: 'P', label: 'Pending' }
```

Comment:
I will change it to be "Unknown", and the value in the table to "U" . We tried using null at first, but it caused some errors because of the way that the javascript treated null, and when we changed it to "P" these errors were no longer there, so we kept it with "P", but I can change it to "U" without any problems.